.github/workflows/tag.yml

```diff
@@ -0,0 +1,16 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
Looks like the `github-tag-action` recommends to set the fetch depth of the checkout for looking for the version message, thoughts?

Reply:
Ah okay, that is good to know. So then we don't need it 👏🏼